### DISCUSSION TITLE
POST to InfluxDB v2 not working

### DISCUSSION THREAD

**Discussion by Tellicious** [RavenSystem/esp-homekit-devices#2159] [MEPLHAA Scripts]
Hello José,

I'm trying to write to an InfluxDB v2 with token authentication, using a command similar to `curl -X POST 'http://192.168.1.2:8086/write?db=testBucket' --header 'Authorization: Token eZQKdDj1esKJNoe5vgzBWESgbKPbIkHIQXQxp_Ntcn0t79YPhQoOo7ssAiH8gRv0oD4XyVChG6tCUGge87jM7A==' --data-binary 'sensor,location=office temperature=20.230'` (tested and working)

I configured a wildcard action in HAA as 
```json
"y0":[
        {"v":-100,
        "r":1,
        "0":{
            "h":[{
               "h":"192.168.1.2",
               "p":8086,
               "m":2,
               "u":"write?db=testBucket",
               "e":"Authorization: Token eZQKdDj1esKJNoe5vgzBWESgbKPbIkHIQXQxp_Ntcn0t79YPhQoOo7ssAiH8gRv0oD4XyVChG6tCUGge87jM7A==",
               "c":"sensor,location=studio temperature=#HAA@0000"
               }]
            }
         }
     ]
```
however the DB doesn't received the data

Looking at the log (services 14 and 15), it seems that the string `Content-length: xx` gets appended to the header. Is it just in the log or is it also sent?

I'm running HAA 12.9.1 on an ESP32-S2

[Nov 19 195055 HAA Log.log](https://github.com/RavenSystem/esp-homekit-devices/files/13404430/Nov.19.195055.HAA.Log.log)

**Comment by RavenSystem** [comment]
You must add `\r\n` to the end of each header line. It is part of how HTTP works:

`"e":"Authorization: Token eZQKdDj1esKJNoe5vgzBWESgbKPbIkHIQXQxp_Ntcn0t79YPhQoOo7ssAiH8gRv0oD4XyVChG6tCUGge87jM7A==\r\n"`

**Reply by Tellicious** [reply]
Thanks, you're right, I thought it was being added by HAA
